src/main/java/com/example/oddleweatherapi/dto/WeatherDTO.java

```diff
@@ -3,15 +3,23 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.validation.constraints.NotBlank;
+
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 public class WeatherDTO {
+    //TODO: NotBlank -> Validation String Boot
     private Integer id;
+
+    @NotBlank
     private String temp;
+
     private String tempMin;
     private String tempMax;
     private String todayDate;
-//    private String weatherType;
-//    private String city;
+    private String weatherName;
+    private String cityName;
+    private Integer weatherType;
```

Comment:
better naming convention,(ex:  weatherTypeId and cityId)
could create a CityDTO and WeatherTypeDTO